### PR TITLE
[Qt] Update packages to 5.15.1

### PR DESCRIPTION
**Pull request by vincowl**

### CODE REVIEW

File: packages/qttools.rb [hunk 1]
```diff
@@ -36,7 +28,6 @@ def self.install
     FileUtils.cp_r 'examples', "#{CREW_DEST_PREFIX}/share/Qt-5"
     FileUtils.cp_r 'include', "#{CREW_DEST_PREFIX}/share/Qt-5"
     FileUtils.cp_r 'mkspecs', "#{CREW_DEST_PREFIX}/share/Qt-5"
-    FileUtils.cp_r 'plugins', "#{CREW_DEST_PREFIX}/share/Qt-5"
```
Comment (uberhacker) [inline packages/qttools.rb:39]: Why no plugins?

File: packages/qtx11extras.rb [hunk 2]
```diff
@@ -28,8 +20,6 @@ def self.build
   end
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}"
-    system "cp -a lib/* #{CREW_DEST_LIB_PREFIX}"
```
Comment (uberhacker) [inline packages/qtx11extras.rb:32]: Why no libraries?

File: packages/qtbase.rb [hunk 3]
```diff
@@ -50,7 +44,8 @@ def self.build
            '-system-zlib',
            '-system-libpng',
            '-system-libjpeg',
-           '-system-freetype'
+           '-system-freetype',
+           '-no-openssl'
```
Comment (uberhacker) [inline packages/qtbase.rb:48]: I'm hesitant to remove openssl.  I'm sure we can figure out a way to include it either by updating libressl or reverting back to openssl.

File: packages/qttools.rb (hunk 1 above)
Comment (vincowl) [reply packages/qttools.rb:39]: This folder doesn't exist anymore. It is not part of the tar file and not produced by the compilation step

File: packages/qtx11extras.rb (hunk 2 above)
Comment (vincowl) [reply packages/qtx11extras.rb:32]: It is not part of the tar file and not produced by the compilation step

File: packages/qtbase.rb (hunk 3 above)
Comment (vincowl) [reply packages/qtbase.rb:48]: Totally agree. But I had to remove it to make the other thing work. Updating libressl or something like it is a huge work due to its impact on the chromebrew basis. As said in a related issue, I chose to remove it in a first step, then state on libressl vs openssl and their updates and finally reactivate it in qtbase

Comment (uberhacker) [reply packages/qtbase.rb:48]: Fair enough.  I'm glad you were able to compile these qt packages successfully because I couldn't do it.

**Pull request merged**